### PR TITLE
add new tutorial: RetroSynthesis Analysis

### PR DESCRIPTION
**Pull request by kenza12**
Add new tutorial for Retrosynthesis Analysis in Synthetic Biology topic.
A video of Retrosynthesis training was also submitted yesterday to GTN video library: https://filesender.renater.fr/?s=download&token=dfb3e2b9-54d6-4b7c-ae95-976432cceda4
Thank you !

### CODE REVIEW

**Comment by gallardoalba** [comment]
Hi @kenza12,
it seems that one of the tools is not available in any of the Galaxy public instance; would be possible to open this PR in order to include it in IUC? I can try to help you with that.

File: topics/synthetic-biology/tutorials/retrosynthesis_analysis/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+Galaxy-SynBioCAD portal provides a set of Retrosynthesis tools aimed at finding pathways to synthesize heterologous compounds in chassis organisms (**RetroRules** ({% cite Duigou2018 %}), **RetroPath2.0** ({% cite Delpine2018 %}), **RP2Paths**, **rpCompletion**).
+
+Retrosynthesis is a concept originally proposed for synthetic chemistry where chemists have to work backwards, starting from a target product to reach precursors that are endogenous to the chassis (host organism).
+
+Typically, the target compound, also named “source compound” is the compound of interest one wishes to produce, while the precursors are usually compounds that are natively present in a chassis strain.
+
+<center><img src="../../images/chassis_target.png" width="500" alt="Chassis - target"/></center>
+
+In this tutorial, we want to obtain the reactions producing the lycopene (source) into the iML1515 Escherichia Coli strain (chassis).
```
Comment (gallardoalba) [inline topics/synthetic-biology/tutorials/retrosynthesis_analysis/tutorial.md:37]:
```suggestion
In this tutorial, we want to obtain the reactions producing the lycopene (source) into the iML1515 _Escherichia Coli_ strain (chassis).
```

File: topics/synthetic-biology/tutorials/retrosynthesis_analysis/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+>    > ### {% icon comment %} How to choose a right diameter ?
+>    >
+>    > The diameter of the sphere including the atoms around the reacting center.
+>    > The higher is the diameter, the more specific are the rules.
+>    {: .comment}
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. A low diameter selects specific rules?
```
Comment (gallardoalba) [inline topics/synthetic-biology/tutorials/retrosynthesis_analysis/tutorial.md:145]:
```suggestion
> 1. Does a low diameter select specific rules?
```

**Comment by kenza12** [comment]
> Hi @kenza12, it seems that one of the tools is not available in any of the Galaxy public instances; would be possible to open this PR in order to include it in IUC? [galaxyproject/tools-iuc#4456](https://github.com/galaxyproject/tools-iuc/pull/4456) I can try to help you with that.

Hi @gallardoalba ,
Thanks for the review. Selenzy tool is already available here :  https://toolshed.g2.bx.psu.edu/view/tduigou/selenzy/34a9d136a5bf and it's a part of another tutorial (https://training.galaxyproject.org/training-material/topics/synthetic-biology/tutorials/basic_assembly_analysis/tutorial.html). Anyway, as I said in this PR (https://github.com/galaxyproject/tools-iuc/pull/4456), the developer will do the necessary changes in the code source. Then, I will modify the wrapper hoping that the tests will pass in IUC next time.
Thank you for your help.

**Comment by gallardoalba** [comment]
> 

Yes sorry, I realized later that it is part of a different tutorial.

**Comment by shiltemann** [comment]
Thanks for the tutorial @kenza12!  And thanks for reviewing @gallardoalba! This looks great to me from a technical point of view. I am uploading the video to the video library now, then I think we can merge after the last minor review items are addressed :) :tada:

**Comment by shiltemann** [comment]
Hi @kenza12! Here is the link to your captions document: https://docs.google.com/document/d/1Zt4gb77Id3IDNNGpmYLFjZ0ruMpPXaSpZSAAWm4Q5KQ/edit?usp=sharing

Let me know when you are done editing it, then I will upload them to YouTube

Thanks!

**Review by shiltemann — APPROVED** [review]

**Comment by kenza12** [comment]
Thanks a lot @shiltemann and all Galaxy team !

**Comment by kenza12** [comment]
Just one last thing, I didn't have the chance to finish correcting the captions.
It's about this sentence:

`We can also visualise the set of pathways **at a glance** by using the SynBioCAD visualisation tool. By clicking on the icon, the set of pathways appears and one can interact with it.`

Before it was:

`We can also visualise the set of pathways **at the clients** by using the SynBioCAD visualisation tool.`

Sorry, but I thought you will wait until I highlight in green. I had to check with my colleague the right word used.

**Pull request merged**